workspaces/confluence/package.json

```diff
@@ -43,7 +43,7 @@
     "@playwright/test": "^1.32.3",
     "@spotify/prettier-config": "^12.0.0",
     "concurrently": "^8.0.0",
-    "lerna": "^7.3.0",
+    "lerna": "^8.0.0",
```

Comment:
@fjudith do we really need this? I'm wondering if we can simply remove it?

Reply:
I've created a PR to do this https://github.com/backstage/community-plugins/pull/872